scripts/qa_crowbarsetup.sh

```diff
@@ -1665,6 +1665,24 @@ function custom_configuration()
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value keystone default "['deployment']['keystone']['elements']['keystone-server']" "['cluster:$clusternameservices']"
             fi
+            if [[ $want_ldap ]] ; then
```

Comment:
i think when we add crowbar_batch support soon, this function will not be needed anymore.